node/src/components/storage/tests.rs

```diff
@@ -411,7 +411,7 @@ fn can_retrieve_store_and_load_deploys() {
     // Also ensure we can retrieve just the header.
     let response = harness.send_request(&mut storage, |responder| {
         StorageRequest::GetDeployHeaders {
-            deploy_hashes: smallvec![*deploy.id()],
+            deploy_hashes: Box::new(smallvec![*deploy.id()]),
```

Comment:
This is not good, we're adding an unnecessary layer of indirection here. A `SmallVec` is conceptually already heap allocated like a `Vec` and a `Box<Vec<_>>` does make little sense in many cases.

It seems our chosen `SmallVec` is actually too big at what I assume is a size of `2`. Try reducing the size of the `SmallVec` instead, if a size of `1` would actually fit, or replace it with a plain `Vec`, which will always be 24 bytes.

Reply:
Resolved by changing from `Mutiple` to `Vec`